app/models/manageiq/providers/vmware/infra_manager.rb

```diff
@@ -200,7 +200,8 @@ def self.default_blacklisted_event_names
     end
 
     def verify_credentials(auth_type = nil, _options = {})
-      self.class.raw_connect(:use_broker => false, :ems => self)
+      user, pwd = auth_user_pwd(auth_type)
```

Comment:
Instead of handling `:ems` I just kept the args consistent with the other callers of raw_connect